Adding a `used-types` rule (#1058)
gajus/eslint-plugin-jsdoc: .README/README.md

|:heavy_check_mark:|:wrench:|[empty-tags](./docs/rules/empty-tags.md#readme)|Checks tags that are expected to be empty (e.g., `@abstract` or `@async`), reporting if they have content|
|:heavy_check_mark:||[implements-on-classes](./docs/rules/implements-on-classes.md#readme)|Prohibits use of `@implements` on non-constructor functions (to enforce the tag only being used on classes/constructors)|
|||[informative-docs](./docs/rules/informative-docs.md#readme)|Reports on JSDoc texts that serve only to restart their attached name.|

> |||[mark-used](./docs/rules/mark-used.md#readme)|Marks all types referenced in `{@link}`d tags as used.|

Author: Totally - good catch. I had the `link` as my primary objective when writing that. I'll rephrase:
```suggestion
|||[mark-used](./docs/rules/mark-used.md#readme)|Marks all types referenced from jsdoc tags as used.|
```

> |||[mark-used](./docs/rules/mark-used.md#readme)|Marks all types referenced in `{@link}`d tags as used.|

Reviewer: Sure. If you could also do so where the other docs mention it too.
